src/components/button/templates/component/template.js

```diff
@@ -108,8 +126,14 @@ export function componentTemplate({ props } : { props : Object }) : string {
 
     let labelTag = '';
 
+    // tag content below button allowed only if fundingicons is false
     if (!props.style.fundingicons) {
-        labelTag = conf.tagline && content[`${label}_tag`] ? content[`${label}_tag`] : '';
+        labelTag = conf.tagline && content[`${conf.tagcontent}`] ? content[`${conf.tagcontent}`] : '';
+    }
+
+    // Hack for buy now branded option to support custom css
+    if (props.style.branded && label === 'buynow') {
```

Comment:
Sure -- I just feel it makes more sense to have an additional class, rather than overloading the `buynow` label class.